website/snippets/_active-auto-exposures-overview.md

```diff
@@ -0,0 +1,29 @@
+
+
+The following table summarizes the differences between visualizing and orchestrating downstream exposures:
+
+| Feature | Visualize downstream exposures | Orchestrate downstream exposures <Lifecycle status="beta"/> |
```

Comment:
The beta pill needs another color or an outline. What did we decicde?

Reply:
I might remove the word "Feature" from the table heading. I was reading anything under it in the 1st column as a feature.

Reply:
yep - we decided on a border so i need to create a separate pr for that!

good shout on the feature column!